MiEdificio/app/src/main/java/com/miedificio/miedificio/networking/RetrofitHelper.java

```diff
@@ -0,0 +1,41 @@
+package com.miedificio.miedificio.networking;
+
+import android.util.Log;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+
+import okhttp3.Interceptor;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class RetrofitHelper {
+
+    public static ObjectMapper buildObjectMapper() {
+        ObjectMapper objectMapper = new ObjectMapper();
+
+        objectMapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
+
+        return objectMapper;
+    }
+
+    static class LoggingInterceptor implements Interceptor {
```

Comment:
Antes que usar un interceptor custom que no loggea toda la info (por ejemplo headers o contenido del response) podemos usar:

compile 'com.squareup.okhttp3:logging-interceptor:3.0.1'

Lo seteas a nivel de OkHttpClient por ejemplo:

        HttpLoggingInterceptor httpLoggingInterceptor = new HttpLoggingInterceptor();
        httpLoggingInterceptor.setLevel(HttpLoggingInterceptor.Level.BODY);

        // TODO: custom timeouts to support the slow response times of the staging server
        // We should reconsider this timeouts before deploying the app
        OkHttpClient.Builder builder = new OkHttpClient.Builder()
                .connectTimeout(40, TimeUnit.SECONDS)
                .readTimeout(40, TimeUnit.SECONDS)
                .writeTimeout(15, TimeUnit.SECONDS);

        if (BuildConfig.DEBUG) {
            builder.addInterceptor(httpLoggingInterceptor);
        }

        OkHttpClient okHttpClient = builder.build();